contracts/lease/src/lease/dto.rs

```diff
@@ -1,19 +1,17 @@
 use serde::{Deserialize, Serialize};
 
-use finance::liability::Liability;
 use oracle::stub::OracleRef;
 use sdk::cosmwasm_std::Addr;
 use timealarms::stub::TimeAlarmsRef;
 
-use crate::{api::LeaseCoin, loan::LoanDTO};
+use crate::{loan::LoanDTO, position::dto::PositionDTO};
```

Comment:
please change the way `PositionDTO` is exported to enable the same usage as for `LoanDTO`, i.e. `position::PositionDTO`